scripts/docs_config/introduction.rst

```diff
@@ -0,0 +1,101 @@
+==============
+ Introduction
+==============
+
+Motivation
+==========
+
+The amount of data associated with modern workloads that need to be processed 
+by modern workloads is continuously growing. To address the increasing demand 
+memory subsystem of modern server platforms is becoming heterogeneous. For 
+example, High-Bandwidth Memory (HBM) introduced in Sapphire Rapids addresses 
```

Comment:
Use the official CPU name as well. Also, HBM has been reintroduced: https://colfaxresearch.com/knl-mcdram/